client/coral-embed-stream/src/reducers/index.js

```diff
@@ -3,11 +3,13 @@ import embed from './embed';
 import configure from './configure';
 import stream from './stream';
 import profile from './profile';
+import debug from './debug';
 
 export default {
   login,
   embed,
   configure,
   stream,
   profile,
+  debug,
```

Comment:
Why do we need another reducer for `debug`. Don't we have a reducer in  `config` already with the `debug` value?